ui/apps/platform/cypress/fixtures/auth/mypermissionsMinimalAccess.json

```diff
@@ -1,7 +1,7 @@
 {
     "resourceToAccess": {
         "Access": "NO_ACCESS",
-        "Administration": "NO_ACCESS",
+        "Administration": "READ_ACCESS",
```

Comment:
This change is because of `Config` which has read access.
Now allowed by `Administration` are the services that used to be tied to the `DebugLogs` permission, and which could allow data leaks. Keeping NO_ACCESS here could also be a good idea.